infrastructure/database/templates/03_setup_tables.template.sql

```diff
@@ -99,15 +98,15 @@ CREATE TABLE "api"."issue_types" (
 );
 
 
-CREATE TABLE "api"."issues" (
-	"id" int4 NOT NULL,
-	"issue_type_id" int4 NOT NULL,
+CREATE TABLE "public"."issues" (
+	"id" serial,
+	"issue_type_id" int4 NOT NULL REFERENCES public.issue_types (id),
 	"created_at" timestamptz NOT NULL DEFAULT now(),
-	"tree_id" text NOT NULL,
+	"tree_id" text NOT NULL REFERENCES public.trees (id),
 	PRIMARY KEY ("id")
 );
 
-CREATE TABLE api.weather_stations (
+CREATE TABLE public.weather_stations (
     id   BIGINT PRIMARY KEY,
     von_datum     DATE,
```

Comment:
why in german here?

Reply:
that was given by the original data source.
we can rename that